server/views/pages/functionalSkills/index.njk

```diff
@@ -26,97 +27,44 @@
-                </tbody>
-              </table>
-            </div>
-          </div>
+          {% if englishSkills.length > 0 %}
+            {{ sortableTable({
+              id: "english-skills",
+              assessmentType: englishSkills
+            }) }}
+          {% else %}
+            <p class="govuk-body">No English functional skill assessment scores recorded in Curious.</p>
+          {% endif %}
 
-          <div class="govuk-summary-card">
-            <div class="govuk-summary-card__title-wrapper">
-              <h2 class="govuk-summary-card__title">Assessment history</h2>
-            </div>
-            <div class="govuk-summary-card__content">
-              <table class="govuk-table" id="functional-skills-assessments-history-table">
-                <caption class="govuk-table__caption govuk-visually-hidden">Details of functional skills assessment history</caption>
-                <thead class="govuk-table__head">
-                  <tr class="govuk-table__row">
-                    <th scope="col" class="govuk-table__header">Subject</th>
-                    <th scope="col" class="govuk-table__header">Assessed on</th>
-                    <th scope="col" class="govuk-table__header">Level</th>
-                    <th scope="col" class="govuk-table__header">Assessment type</th>
-                    <th scope="col" class="govuk-table__header">Source</th>
-                  </tr>
-                </thead>
+          <h2 class="govuk-heading-m">Maths functional skills assessment scores</h2>
 
-                <tbody class="govuk-table__body">
-                {% for assessment in allFunctionalSkills.assessments %}
-                  <tr class="govuk-table__row">
-                    <td class="govuk-table__cell">{{ assessment.type | formatFunctionalSkillType }}</td>
-                    {% if assessment.assessmentDate and assessment.grade %}
-                      <td class="govuk-table__cell">{{ assessment.assessmentDate | formatDate('D MMMM YYYY') }}</td>
-                      <td class="govuk-table__cell">{{ assessment.grade }}</td>
-                      <td class="govuk-table__cell">
-                        Initial
-                      </td>
-                      <td class="govuk-table__cell">
-                        <span class="govuk-hint">Curious</span>
-                      </td>
-                    {% else %}
-                      <td class="govuk-table__cell" colspan="4">No functional skill assessment scores recorded in Curious</td>
-                    {% endif %}
-                  </tr>
-                {% endfor %}
-                </tbody>
-              </table>
-            </div>
-          </div>
+          {% if mathsSkills.length > 0 %}
+            {{ sortableTable({
+              id: "maths-skills",
+              assessmentType: mathsSkills
+            }) }}
+          {% else %}
+            <p class="govuk-body">No Maths functional skill assessment scores recorded in Curious.</p>
```

Comment:
Empty states weren't included in the JIRA ticket, so I've kept the format from the previous design, but included the skill name in each!